src/Converter.php

```diff
@@ -27,9 +26,9 @@
 
 class Converter
 {
+
```

Comment:
There are a lot of unnecessary whitespace/newline changes in this file. This affects readability, increases the surface area of the PR and makes reverts/cherry-picks harder - can we please reduce these to what's necessary?

Reply:
Done